components/channel_notifications_modal/index.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+
+import ChannelNotificationsModal from './channel_notifications_modal.jsx';
+
+function mapStateToProps(state, ownProps) {
+    const config = state.entities.general.config;
+    const sendPushNotifications = config.SendPushNotifications === 'true';
+
+    return {
+        ...ownProps,
```

Comment:
We actually don't need to include `...ownProps` in this return statement, the connect function handles it. I know we do it everywhere but it was a mistake that got copied/pasted everywhere before we caught it. @stephenkiers has a a PR #798 to remove them all

I'm 0/5 if you want to go through the effort of removing them here though, I'm trying to avoid being too nitpicky because of the volume of PRs coming through

Reply:
Ah, fascinating! I'll either rip them out as part of the PRs, or go back and fix things up later depending on the volume of feedback I receive.